stable/artifactory-ha/templates/artifactory-node-pdb.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if gt (int .Values.artifactory.node.replicaCount) 0 }}
```

Comment:
Not sure we should do it like this. Many times, we scale the StatefulSet directly and not via the helm chart.
If no errors, please leave this PDB.

Reply:
This is removed to the old code.